config/settings/new_staging.yml

```diff
@@ -0,0 +1,11 @@
+dfe_signin:
+  issuer: https://signin-test-oidc-as.azurewebsites.net
+  profile: https://signin-test-pfl-as.azurewebsites.net
+  secret: please_change_me # Override with SETTINGS__DFE_SIGNIN__SECRET
```

Comment:
remember to change it

Reply:
Thanks. It's already set in the new staging env :) 👍